src/transaction/components/Transaction.tsx

```diff
@@ -7,11 +7,15 @@ export function Transaction({
   className,
   children,
   contracts,
+  chainId,
+  capabilities,
```

Comment:
Keep alphabetic order. 

Btw in other places we have `chain` and not `chainId`, also why this should override the chain set at OnchainProvider level?

Reply:
Updated to alphabetic